MonoGame.Framework/Graphics/GraphicsDebug.cs

```diff
@@ -2,10 +2,35 @@
 // This file is subject to the terms and conditions defined in
 // file 'LICENSE.txt', which is part of this source code package.
 
+using System;
+
 namespace Microsoft.Xna.Framework.Graphics
 {
-    public partial class GraphicsDebug
+    public partial class GraphicsDebug : IDisposable
     {
+        private readonly GraphicsDevice _device;
+        private bool _isDisposed = false;
+
+        /// <summary>
+        /// Constructs a new instance of <see cref="GraphicsDebug"/>, which provides debugging APIs
+        /// for the underlying graphics hardware.
+        /// </summary>
+        /// <param name="device">The associated graphics device.</param>
+        public GraphicsDebug(GraphicsDevice device)
```

Comment:
Should this constructor be public at all?  Since it is only created within `GraphicsDevice`, it could be `internal` keeping users from thinking they have to allocate one themselves.  Also in that case I wouldn't even document it as it is a private implementation detail.